contrib/podmanimage/stable/Dockerfile

```diff
@@ -11,16 +11,12 @@ FROM fedora:latest
 # Don't include container-selinux and remove
 # directories used by yum that are just taking
 # up space.
-RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install podman fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
+RUN useradd podman; yum -y update; yum -y reinstall shadow-utils; yum -y install podman fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
```

Comment:
I don't think "build" should change here to podman....

Reply:
ditto the others.

Reply:
Why not?

Reply:
I assumed "build" was expected to be in the passwd file and this might mess something up that was dependent on it.  I'm not sure where that dependency is though, or if we even need to do any useradd here.  I don't recall the genesis of it.

Reply:
and, quick little innocuous changes like this have the tendency to come back and bite me later.

Reply:
build is just the user that you use.  What I am trying to do is build a container image that can be used both by root and as rootless.

podman run --user podman quay.io/stable/podman podman info
and
podman run  quay.io/stable/podman podman info

Would work.

Having the user be "build" makes no sense, (It does for Buildah though).  This was a cut and paste from the buildahimage, and I am just fixing it.